ord_schema/message_helpers.py

```diff
@@ -316,17 +322,43 @@ def load_message(filename, message_type, input_format):
         ValueError: if the message cannot be parsed, or if `input_format` is not
             supported.
     """
-    if input_format == 'binary':
+    input_format = MessageFormats(input_format)
+    if input_format == MessageFormats.BINARY:
         mode = 'rb'
     else:
         mode = 'r'
     with open(filename, mode) as f:
-        if input_format == 'json':
-            return json_format.Parse(f.read(), message_type())
-        if input_format == 'pbtxt':
-            return text_format.Parse(f.read(), message_type())
-        if input_format == 'binary':
-            message = message_type()
-            message.ParseFromString(f.read())
+        try:
+            if input_format == MessageFormats.JSON:
+                message = json_format.Parse(f.read(), message_type())
```

Comment:
Stylistically, is this preferred to `return` on every line? It is slightly more readable.

Reply:
I know...but pylint was giving me an error about the `return` statements not all returning an expression so I just went with this since it wasn't obvious how to fix that error.

I'll just disable the error b/c I agree returns are more readable.